### PR TITLE
MDN CSS: Fix duplicate entries

### LINKED ISSUE #580: MDN CSS: Update parser to prevent duplicate articles
### Description

After running the Fathead Test, it shows that our `output.txt` file contains 10 duplicate entries:

```
# 10 DUPLICATE ARTICLES FOUND:
# -webkit-appearance        duplicated on lines 155, 156
# left                      duplicated on lines 770, 771
# -moz-placeholder          duplicated on lines 67, 68
# position                  duplicated on lines 949, 950
# -webkit-perspective       duplicated on lines 195, 196
# perspective property      duplicated on lines 940, 941
# symbols                   duplicated on lines 1060, 1061
# right                     duplicated on lines 990, 991
# color                     duplicated on lines 566, 567
# perspective               duplicated on lines 938, 939
```

These need to be reviewed  to determine if the duplicate titles have duplicate abstracts, and if so we need to update the parser to ensure we don't create any duplicate entries. If the abstracts are different it means we need to update the parser to create disambiguations entries for these entries. We will also need a way to differentiate the titles so all the article titles are unique.

## Get Started
- [x] 1) Claim this issue by commenting below
- [ ] 2) Review our [Contributing Guide](https://github.com/duckduckgo/zeroclickinfo-fathead/blob/master/CONTRIBUTING.md)
- [ ] 3) [Set up your development environment](https://docs.duckduckhack.com/welcome/setup-dev-environment.html), and fork this repository
- [ ] 4) Create a Pull Request

## Resources
- Join [DuckDuckHack Slack](https://quackslack.herokuapp.com/) to ask questions
- Join the [DuckDuckHack Forum](https://forum.duckduckhack.com/) to discuss project planning and Instant Answer metrics
- Read the [DuckDuckHack Documentation](https://docs.duckduckhack.com/) for technical help

---

Instant Answer Page: https://duck.co/ia/view/mdn_css

### PR DESCRIPTION
**Pull request by hchienjo**
## Description of new Instant Answer, or changes

1. Eliminate duplicate entries in output.txt
2. Ensures there are no blank abstracts.
3. Ensures all -webkit and -moz that redirect to same article are written correctly.
4. Remove autogenerated redirect_urls.yml file from repo.
5. Process mozilla links exclusively.

## Related Issues and Discussions

Fixes #580 

## People to notify

@duckduckgo/community-leaders 

## Testing & Review
To be completed by Language Leader (or DDG Staff)

**Pull Request**
- [x] Title follows correct format (Specifies Instant Answer + Purpose)
- [x] Description contains a valid Instant Answer Page Link (e.g. https://duck.co/ia/view/my_ia)

**Instant Answer Page**
- [x] Instant Answer page is correctly filled out and contains:
    - One topic for the Search Space Language (Java, Python, Scala, Ruby, etc.)
    - One topic from: Reference, Help, Libraries, Tools
        - Documentation Fatheads are considered "Reference"
    - Description, Data source, and 2+ example queries
    - Perl Module (e.g. "DDG::Fathead::PerlDoc" -- we only need a name, not an actual file)
    - Source Name (for "More at <source_name>" link)
    - Source Domain (must contain http:// or https:// -- can be the same as Data Source)
    - Source Info (used as Subtitle for each Article -- usually matches the IA Name)
    - 'Skip Abstract' is checked off
    - Source ID (ping @moollaza to assign one, once Fathead is ready for Beta deploy)
- [ ] Fathead Tests are passing (run `$ duckpan test <fathead_id>`)
    - Tester should report any failures

**Code**
- [x] Uniformly indented, well commented
- [ ] Fetch.sh and Parse.sh run without errors
- [x] Output contains no blank lines, or multi-line entries

**Pull Request Review Guidelines**: https://docs.duckduckhack.com/programming-mission/pr-review.html

---

Instant Answer Page: https://duck.co/ia/view/mdn_css

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by moollaza** [comment]
@hchienjo Thanks! This LGTM :+1:

File: lib/fathead/mdn_css/fetch.pl [hunk 1]
```diff
@@ -57,12 +57,15 @@
...
                           "$file_number.html";
                         ++$file_number;
-                        
-                        #if the keyword URL redirects to another URL, write the keywords to a file
-                        if($url ne $tx->req->url) {
-                            my $keyword  = substr ($url, rindex($url, "/") + 1);
-                            my $redirect = substr ($tx->req->url, rindex($tx->req->url, "/") + 1);
-                            $redirect_map{$redirect} = $keyword;
+
+      #if the keyword URL redirects to another URL, write the keywords to a file
```
Comment (moollaza) [inline lib/fathead/mdn_css/fetch.pl]: nit: looks like indentation is off for this comment?

Comment (hchienjo) [inline lib/fathead/mdn_css/fetch.pl]: I will look into it. Could be caused by Perltidy as I have set it such that it runs after each save.

**Pull request merged**